spec/support/fixtures/approvals/email_subscriptionsunsubscribe.approved.html

```diff
@@ -7,11 +7,11 @@
   <body>
     <div class="unsubscribed">
   <div class="unsubscribed__title">
-    You have been unsubscribed from DEV digest emails. 😔
+    You have been unsubscribed from DEV(local) digest emails. 😔
   </div>
   <br /><div class="unsubscribed__message">
     You will no longer receive emails like this. If you'd like to re-subscribe or manage your full email preferences, please
-    <a href="https://dev.to/settings/notifications">click here</a>.
+    <a href="http://localhost:3000/settings/notifications">click here</a>.
```

Comment:
I'm not 100% sure that this is going to work when we build travis from github, and I'm not sure this is the best way to do it. May need to find another solution